pkg/webui/console/containers/side-bar/index.js

```diff
@@ -55,39 +68,41 @@ const Sidebar = ({ isDrawerOpen }) => {
 
   const sidebarClassnames = classnames(
     style.sidebar,
-    'd-flex pos-fixed direction-column j-between gap-cs-m bg-tts-primary-050',
+    'd-flex direction-column j-between gap-cs-m bg-tts-primary-050',
     {
       [style.sidebarMinimized]: isMinimized,
-      [style.sidebarOpen]: isMobile && isDrawerOpen,
-      'p-cs-s': !isMinimized,
-      'p-vert-cs-m': isMinimized,
+      [style.sidebarOpen]: isDrawerOpen,
+      'p-cs-m': !isMinimized,
+      'p-vert-cs-s': isMinimized,
       'p-sides-cs-xs': isMinimized,
     },
   )
 
   return (
-    <div className={sidebarClassnames} id="sidebar-v2">
-      <SidebarContext.Provider value={{ topEntities, onMinimizeToggle, isMinimized }}>
-        <div className="d-flex direction-column gap-cs-m">
-          <SideHeader />
-          <div>
-            <SwitcherContainer />
-            <SearchButton />
+    <>
+      <div className={sidebarClassnames} id="sidebar">
+        <SidebarContext.Provider
+          value={{ topEntities, onMinimizeToggle, isMinimized, onDrawerCloseClick }}
+        >
+          <div className="d-flex direction-column gap-cs-l">
+            <SideHeader />
+            <div className="d-flex direction-column gap-cs-m">
+              <SwitcherContainer />
+              <SearchButton onClick={() => null} />
+            </div>
+            <SidebarNavigation />
           </div>
-          <SidebarNavigation />
-        </div>
-        <SideFooter />
-      </SidebarContext.Provider>
-    </div>
+          <SideFooter />
+        </SidebarContext.Provider>
+      </div>
+      <div className={style.sidebarBackdrop} onClick={onDrawerCloseClick} />
```

Comment:
Note that the backdrop was missing in the earlier implementation. There was only a box-shadow, which however was not according to the screen design.